### DISCUSSION TITLE
Daily 7/11

### DISCUSSION THREAD

**Discussion by rpgrca** [rpgrca/proyecto-ispc-fullstack#279] [General]
Ayer hicimos un balance de lo que faltaba por hacer así que terminé el diagrama de clases (voy a agregarlo a la documentación como un archivo de imagen también ya que es muy grande), agregué a Artículos que pueda hacer modificaciones, listados y bajas para completar nuestro CRUD, actualicé la documentación del diseño del sistema y agregué una guía rápida para instalar y correr el sistema. Hoy durante el día voy a armar los casos de uso y agregarlos a la Wiki y a la documentación de diseño. A la noche voy a estar en la clase de full-stack y tal vez grabe lo que sería mi parte para el video grupal.

**Comment by NataliaAlvarezIspc** [comment]
El viernes participé en la clase de Inglés y estuve terminando con el portfolio personal, luego lo subí al foro para presentarlo. El fin de semana estuve haciendo las correcciones que pidieron en el formulario de registro y las subí al repositorio. También grabé el video para la presentación final. Ayer realizamos nuestra reunion semanal de grupo en la que estuve presente. 
Hoy estuve repasando un  poco de Inglés y empecé a agregarles a algunas páginas el enlace a la página de Almoneda, trataré de terminar todo para mañana ya subirlo al repositorio

**Comment by Brian-Almada** [comment]
Hoy estuve haciendo tareas de ingles. Ahora viendo lo del video y lo de wordpress.

**Comment by AhumadaCeleste** [comment]
Ayer estudie para ingles y tambien practique con la clase de el jueves ejecutar algo en java scritp. El fin de semana le dedique tiempo a python. Hoy voy a estudiar ingles
